### ISSUE TITLE
PHP 8.2 deprecation message "Creation of dynamic property"

### ISSUE THREAD

**Issue by michaelarnauts** [vaimo/composer-patches#104]
**Pre-requisites**
1. Use PHP 8.2

**To Reproduce**
1. Install the package

**Actual**
The output of composer install contains this warning:
```
Deprecation Notice: Creation of dynamic property Vaimo\ComposerPatches\Patch\File\Applier::$applierErrorFactory is deprecated in /opt/app/vendor/vaimo/composer-patches/src/Patch/File/Applier.php:63
```

**Comment by hostep** [comment]
Yep, also running into this.
It's just a warning, but still, would be nice to see this fixed soon-ish (Magento 2.4.6 comes with PHP 8.2 support and will be released in about 3 weeks) 🙂 

Thanks!

**Comment by zamoroka** [comment]
@hostep @michaelarnauts  thanks for your contribution. The fix is released in 5.1.1
